backend/node/base.py

```diff
@@ -343,8 +343,8 @@ async def _run_genvm(
                 stderr="",
                 genvm_log=[],
                 result=genvmbase.ExecutionError(
-                    message="Execution timed out",
-                    kind=genvmbase.ResultCode.CONTRACT_ERROR,
+                    message="timeout",
```

Comment:
I will add this "timeout" to autogenerated constants in next release, but it is the correct message, as all VMErrors (previously misleadingly named ContractErrors start with snake case error code)